### PR TITLE
dev-lang/nasm: bump to 2.13.01, EAPI=6, fix doc dependencies

### PR DESCRIPTION
**Pull request by sbraz**
Related bug: https://bugs.gentoo.org/show_bug.cgi?id=619360
I added some missing dependencies and removed texinfo which isn't required.
I will make a rekeywording bug for the arches that do not have the new deps once this is merged.
```diff
--- nasm-2.12.01.ebuild	2017-03-21 23:37:52.370951461 +0100
+++ nasm-2.13.01.ebuild	2017-06-15 22:10:12.276993250 +0200
@@ -1,7 +1,7 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
 inherit flag-o-matic
 
 DESCRIPTION="groovy little assembler"
@@ -10,14 +10,22 @@
 
 LICENSE="BSD-2"
 SLOT="0"
-KEYWORDS="amd64 ~ia64 x86 ~x64-cygwin ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~x64-macos"
+KEYWORDS="~amd64 ~x86 ~x64-cygwin ~amd64-linux ~x86-linux ~x64-macos"
 IUSE="doc"
 
-DEPEND="dev-lang/perl
-	doc? ( app-text/ghostscript-gpl sys-apps/texinfo )"
-RDEPEND=""
+DEPEND="
+	dev-lang/perl
+	doc? (
+		app-text/ghostscript-gpl
+		dev-perl/Font-TTF
+		dev-perl/Sort-Versions
+		media-fonts/clearsans
+		virtual/perl-File-Spec
+	)
+"
 
 S=${WORKDIR}/${P/_}
+DOCS=( AUTHORS CHANGES ChangeLog README TODO )
 
 src_configure() {
 	strip-flags
@@ -25,17 +33,15 @@
 }
 
 src_compile() {
-	emake nasmlib.o
-	emake all
+	default
 	use doc && emake doc
 }
 
 src_install() {
 	emake INSTALLROOT="${D}" install install_rdf
-	dodoc AUTHORS CHANGES ChangeLog README TODO
-	if use doc ; then
-		doinfo doc/info/*
-		dohtml doc/html/*
-		dodoc doc/nasmdoc.*
+	if use doc; then
+		DOCS+=( doc/nasmdoc.{pdf,txt} )
+		HTML_DOCS=( doc/html/. )
 	fi
+	einstalldocs
 }

```

### CODE REVIEW

File: dev-lang/nasm/metadata.xml [hunk 1]
```diff
@@ -15,5 +15,6 @@ Pentium, P6, MMX, 3DNow!, SSE and SSE2 opcodes, and has macro capability.
   </longdescription>
   <upstream>
     <remote-id type="sourceforge">nasm</remote-id>
+	<bugs-to>https://bugzilla.nasm.us/</bugs-to>
```
Comment (SoapGentoo) [inline dev-lang/nasm/metadata.xml]: did you run `xmllint` on this?

File: dev-lang/nasm/nasm-2.13.01.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		app-text/ghostscript-gpl
+		dev-perl/Font-TTF
+		dev-perl/Sort-Versions
+		media-fonts/clearsans
+		virtual/perl-File-Spec
+	)
+"
+
+S=${WORKDIR}/${P/_}
+DOCS=( AUTHORS CHANGES ChangeLog README TODO )
```
Comment (SoapGentoo) [inline dev-lang/nasm/nasm-2.13.01.ebuild]: all of these are part of the default set?

File: dev-lang/nasm/nasm-2.13.01.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+src_compile() {
+	default
+	use doc && emake doc
+}
+
+src_install() {
+	emake INSTALLROOT="${D}" install install_rdf
+	if use doc; then
+		DOCS+=( doc/nasmdoc.{pdf,txt} )
```
Comment (SoapGentoo) [inline dev-lang/nasm/nasm-2.13.01.ebuild]: rather just do `dodoc doc/nasmdoc.{pdf,txt}` here and nix the prior `DOCS` definition

File: dev-lang/nasm/metadata.xml (hunk 1 above)
Comment (sbraz) [inline dev-lang/nasm/metadata.xml]: I blame vim! (I didn't see that the original had spaces and my vim config uses tabs for those files, which is Gentoo's default I guess).

**Pull request closed without merging**